pallets/fee/Cargo.toml

```diff
@@ -33,6 +33,7 @@ mocktopus = "0.8.0"
 frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
 pallet-timestamp = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
 currency = { path = "../currency", default-features = false, features = ["testing-constants"] }
+security = { path = "../security", features = ['testing-utils'] }
```

Comment:
Why is this necessary?

Reply:
Because of this
`Security::set_active_block_number(1);`

and declaration is like this
```
/// for testing purposes only!
#[cfg(feature = "testing-utils")]
    pub fn set_active_block_number(n: T::BlockNumber) {
	ActiveBlockCount::<T>::set(n);
}
```